lib/fog/profitbricks/requests/compute/get_all_lans.rb

```diff
@@ -93,6 +93,8 @@ def get_all_lans(datacenter_id)
                         :method  => 'GET',
                         :path    => "/datacenters/#{datacenter_id}/lans?depth=1"
                     )
+                rescue => error
+                    Fog::Errors::NotFound.new(error)
```

Comment:
I am aware of the flaws in the current implementation of error handling - it will be revised and updated on a separate branch.